public/content/documentation/native/controls/dropdown.md

```diff
@@ -0,0 +1,337 @@
+
+**UIKit**
+  - If VoiceOver is not reaching a particular element, set the element's `isAccessibilityElement` to `true`
+    - **Note:** You may need to adjust the programmatic name, role, state, and/or value after doing this, as this action may overwrite previously configured accessibility.
+  - Use `accessibilityViewIsModal` to contain the screen reader focus inside the modal.
+  - To move screen reader focus to newly revealed content, use `UIAccessibility.post(notification:argument:)` that takes in `.screenChanged` and the newly revealed content as the parameter arguments.
+  - To NOT move focus, but dynamically announce new content: use `UIAccessibility.post(notification:argument:)` that takes in `.announcement` and the announcement text as the parameter arguments.
+  - `UIAccessibilityContainer` protocol: Have a table of elements that defines the reading order of the elements.  
+
+**SwiftUI**
+  - For general focus management that impacts both screen readers and non-screen readers, use the property wrapper `@FocusState` to assign an identity of a focus state.
+    - Use the property wrapper `@FocusState` in conjunction with the view modifier `focused(_:)` to assign focus on a view with `@FocusState` as the source of truth.
+    - Use the property wrapper `@FocusState`in conjunction with the view modifier `focused(_:equals:)` to assign focus on a view, when the view is equal to a specific value.
+  - If necessary, use property wrapper `@AccessibilityFocusState` to assign identifiers to specific views to manually shift focus from one view to another as the user interacts with the screen with VoiceOver on.
+
+### Announcement examples
+
+- "Button label, chosen option label, button, collapsed"  (Button that opens dropdown with selected option from dropdown and state)
+- "Selected, chosen option label, button" (Selected option in dropdown list)
+- "Label, button" (Other options in list that populate dropdown)
+- "Dismiss context menu, button"  (Optional Hidden button that closes dropdown)
+
+
+## Android Developer Notes
+
+- A dropdown or spinner is a button that opens a list of options.  When an option is chosen, it displays in the field (replaces the current option or placeholder)
+- The screen reader focus moves directly to first option in dropdown/spinner upon double tapping the dropdown button and is confined in the list.
+- Sometimes, a hidden "dismiss context menu" button after the last item is available to close it.
+- Focus should go back to the triggering dropdown/spinner button, displaying the new option
+- There must be a visible label for the dropdown field that is not a placeholder and it describes the purpose of the dropdown.
+- The screen reader focus also remains confined in the dropdown list
+- The state of expanded or collapsed should be announced
+
+### Name
+- Name describes the purpose of the control, with additional label description if needed.
+
+**Android Views**
```

Comment:
Similar to the iOS "UIKit" and "SwiftUI", "Android Views" and "Jetpack Compose" could have the same applied -- they aren't headings, and there are multiples under each "Name", "Role", "Groupings", "State", and "Focus", I think these should likely stay indented and bulleted as it is in current-day MagentaA11y.